### PR TITLE
Fix issue "Permission denied" issue on Windows, #766

### PR DESCRIPTION
**Pull request by gustavdelius**
@lukaszbednarz diagnosed the bug correctly in #766 and this pull request is a simple way to fix it.

### CODE REVIEW

**Comment by aappling-usgs** [comment]
This fixed the problem for me :+1:

**Comment by hadley** [comment]
Could you please include a unit test so this doesn't occur again?

Can you please add a bullet to NEWS? It should briefly [describe the change](http://style.tidyverse.org/news.html) and end with `(@yourname, #issuenumber)`.

**Comment by JoFAM** [comment]
Any chance to get this fixed @hadley ? I now have to manually build the old roxygen2 so I can install it on my students computers when I teach about packages containing S4. Which is basically the entire Bioconductor suite...

**Comment by hadley** [comment]
I'lll try and get a roxygen2 release ready later this week.

File: NEWS.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # roxygen2 6.1.0.9000
 
 * roxygen2 now specifically imports desc >= 1.2.0 (@crsh, #773, #777, #779)
+* `package_files()` ignores leading whitespace in Collate field (@gustavdelius,
```
Comment (hadley) [inline NEWS.md]: Could you please add a newline above this bullet?

File: R/source.R [hunk 2]
```diff
@@ -6,6 +6,8 @@ package_files <- function(path) {
   collate <- scan(text = desc$Collate %||% "", what = "", sep = " ",
     quiet = TRUE)
 
+  collate <- collate[collate != ""]
+
```
Comment (hadley) [inline R/source.R]: Can you please remove this empty line?

File: NEWS.md (hunk 1 above)
Comment (gustavdelius) [reply NEWS.md]: Done

**Comment by gaborcsardi** [comment]
Btw. just taking `Collate` is still incorrect, because there is also `Collate.windows` and `Collate.unix`  and these  should  be preferred. I think it is just easier to use `desc::get_collate()`, TBH.

**Comment by hadley** [comment]
Ah got it, sorry for making you do unneeded work @gustavdelius 😞

**Comment by lukasz-bednarz-reddeersystems** [comment]
When new roxygen2 with fix will be available in CRAN?

Now new devtools require new roxygen2 6.1.0 and it breaks all our devops.

**Comment by hadley** [comment]
Later this week, all going well.

**Pull request closed without merging**